frame/support/test/tests/construct_runtime.rs

```diff
@@ -142,6 +224,8 @@ frame_support::construct_runtime!(
 		Module1_1: module1::<Instance1>::{Pallet, Call, Storage, Event<T>, Origin<T>},
 		Module2: module2::{Pallet, Call, Storage, Event, Origin},
 		Module1_2: module1::<Instance2>::{Pallet, Call, Storage, Event<T>, Origin<T>},
+		NestedModule3: nested::module3::{Pallet, Call, Storage, Event, Origin},
+		Module3: module3::{Pallet, Call, Storage, Event, Origin},
```

Comment:
you can also test with the `Config` part, I think the PR currently wouldn't work as both pallets would generate the same field in the runtime genesis config.

Reply:
Yep, it doesn't work... that means I'm going to have to do another migration to proc macros.